services/skus/controllers.go

```diff
@@ -693,7 +735,7 @@ func DeleteOrderCreds(service *Service) handlers.AppHandler {
 		}
 
 		isSigned := r.URL.Query().Get("isSigned") == "true"
-		if err := service.DeleteOrderCreds(ctx, id, isSigned); err != nil {
+		if err := service.DeleteOrderCreds(ctx, id, nil, isSigned); err != nil {
```

Comment:
Can we avoid passing nil into methods or functions it leads to nil checks and harder to maintain code

Reply:
Plus a lot to this. This does not look good to me.

Instead, declare that as `uuid.UUID`, and pass `uuid.Nil` to indicate the emptiness of the request id.

Reply:
Updated by the patch #2314.